app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -422,24 +416,39 @@ public void showVisualizeFragment() {
             // move the play button to the middle
 
             shiftAmnt = (mPlaybackToolbar.getWidth() - mPlayPauseButton.getWidth()) / 2;
-            TranslateAnimation animation = new TranslateAnimation(shiftAmnt, 0, 0, 0);
-            animation.setDuration(200);
-            animation.setInterpolator(new AccelerateDecelerateInterpolator());
-            animation.setFillAfter(true);
-            mPlayPauseButton.setTranslationX(-1 * shiftAmnt);
-            mPlayPauseButton.startAnimation(animation);
-
+            movePlayButton(shiftAmnt);
             // move the info off the screen
-            AlphaAnimation alphaAnimation = new AlphaAnimation(1f, 0f);
-            alphaAnimation.setDuration(300);
-            alphaAnimation.setStartOffset(100);
-            alphaAnimation.setFillAfter(true);
-            alphaAnimation.setInterpolator(new AccelerateInterpolator());
-            findViewById(R.id.ll_show_info).startAnimation(alphaAnimation);
+            moveInfo(1f, 0f);
+
         }
     }
 
     /**
+     * Move the play button to the middle
+     */
+
+    public void movePlayButton(float shiftAmnt) {
+        TranslateAnimation animation = new TranslateAnimation(shiftAmnt, 0, 0, 0);
+        animation.setDuration(200);
+        animation.setInterpolator(new AccelerateDecelerateInterpolator());
+        animation.setFillAfter(true);
+
+    }
+
+    /**
+     * Create animation controls the alpha level
+     */
+    public void moveInfo(float fromAlpha, float toAlpha) {
+        AlphaAnimation alphaAnimation = new AlphaAnimation(fromAlpha, toAlpha);
+        alphaAnimation.setDuration(300);
+        alphaAnimation.setStartOffset(100);
```

Comment:
Same here. You can extract this as: `private final int MS_ANIMATION_START_OFFSET = 100;` I used the `MS` prefix here to indicate that it is in milliseconds. If it not, change it accordingly.